test/miniml.py

```diff
@@ -27,3 +30,10 @@ def fit(self, x, y, **kwargs):
 
     def save(self, model_dir):
         test.write_json(self.parameters, model_dir)
+
+    @classmethod
+    def load(cls, model_dir):
+        return cls(**env.read_json(model_dir))
```

Comment:
why splat dict just to assign as dict (at 24)?

Reply:
removed

Reply:
Because kwargs is different than a dict:

from terminal
```
In [1]: def test(**kwargs):
   ...:     pass
   ...:
   ...: test({1:2})
   ...:
   ...:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-e3da7728fca3> in <module>()
      2     pass
      3
----> 4 test({1:2})

TypeError: test() takes 0 positional arguments but 1 was given
```

Reply:
i mean, change the __init__ method to take a dict, and just pass the dict here.